data/mods/Magiclysm/Spells/attunements/Vulcanist.json

```diff
@@ -71,5 +71,51 @@
     "flags": [ "VERBAL", "SOMATIC", "LOUD", "MUST_HAVE_CLASS_TO_LEARN" ],
     "extra_effects": [ { "id": "lava_bomb_shrapnel" }, { "id": "lava_bomb_heat" }, { "id": "lava_bomb_ter" } ],
     "spell_class": "VULCANIST"
+  },
+  {
+    "id": "vulcanist_pyroclastic_flow",
+    "type": "SPELL",
+    "name": "Pyroclastic Flow",
+    "description": "Pulls up the heat of the Earth, along with considerable smoke mixed with ashes.",
+    "effect": "attack",
+    "shape": "line",
+    "valid_targets": [ "hostile", "ground", "ally" ],
+    "damage_type": "heat",
+    "max_level": 35,
+    "difficulty": 10,
```

Comment:
i noticed all of your spells are very high difficulty and was wondering if you thought they needed to be high difficulty because they were attunement spells.

Reply:
in my tests the high magic skill and intelligence gradually reduce spell difficulty, so 5 lvl spell with difficulty 9 has 60% of fail on 5 magic level and 6 intelligence (which is pretty easy to get for endgame spells) (+1 magic skill level decrease it on 8%, when +1 intelligence decrease it on **30%** ). in summary i think its not so big deal for so high level spells on the moment people can get them